instrumentation/elasticsearch/elasticsearch-api-client-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/apiclient/v8_0/ElasticsearchApiClientInstrumentationModule.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.elasticsearch.apiclient.v8_0;
+
+import static java.util.Collections.singletonList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class ElasticsearchApiClientInstrumentationModule extends InstrumentationModule {
+  public ElasticsearchApiClientInstrumentationModule() {
+    super("elasticsearch-client", "elasticsearch-api-client", "elasticsearch");
```

Comment:
i think you should also include `elasticsearch-api-client-8.0` and perhaps drop `elasticsearch-client`?